src/xcode/ENA/ENA/Source/Services/Event Registration/CheckinRiskCalculation/CheckinRiskCalculation.swift

```diff
@@ -45,7 +45,15 @@ final class CheckinRiskCalculation: CheckinRiskCalculationProtocol {
 		// 1. Drop Submitted CheckIns: filter out all CheckIns with Check-in Submitted set to true.
 		// This is to ensure that users don't match their own submitted check-ins.
 
-		let chickens = eventStore.checkinsPublisher.value.filter { !$0.checkinSubmitted }
+		let chickens = eventStore.checkinsPublisher.value.filter {
+			!$0.checkinSubmitted
+		}
+		// 1.1 Filter by age: filter out all CheckIns where the date described by `endTimestamp` is older than `maxCheckInAgeInDays`. The calculation shall use seconds/timestamps for calculation (i.e. `maxCheckInAgeInDays x 86400`)
+		.filter {
+			let maxCheckInAgeInDays = config.presenceTracingParameters.riskCalculationParameters.maxCheckInAgeInDays
+			let checkinIsToOld = Int($0.checkinEndDate.timeIntervalSince1970) < maxCheckInAgeInDays * 86400
```

Comment:
I think the logic is not quite right on the implementation. Wouldn't this compare to the "age in days" in reference to the 1970-01-01? I think we want to know the age in days in respect to right now, so the checkin was e.g. "14 days ago"

Reply:
Good point. Added change.